src/handler/callback_context.jl

```diff
@@ -0,0 +1,46 @@
+using .Contexts
+const CallbackContextItems = Union{Nothing, Vector{NamedTuple}}
+const TriggeredParam = NamedTuple{(:prop_id, :value)}
+mutable struct CallbackContext
+    response::HTTP.Response
+    inputs::Dict{String, Any}
+    states::Dict{String, Any}
+    outputs_list::Vector{Any}
+    inputs_list::Vector{Any}
+    states_list::Vector{Any}
+    triggered::Union{Nothing, Vector{TriggeredParam}}
```

Comment:
Is the `Nothing` necessary? Won't `triggered` at least be a length-0 Vector? Just trying to make sure this is as easy to use as possible, so users don't need fallback code for the `Nothing` case.

Reply:
@alexcjohnson I agree, I'll fix it

Reply:
@alexcjohnson fixed at eb51d4b